src/rules/__tests__/prefer-expect-assertions.test.ts

```diff
@@ -205,6 +205,21 @@ ruleTester.run('prefer-expect-assertions', rule, {
         },
       ],
     },
+    {
+      code: dedent`
+        it("it1", async function() {
+          expect(someValue).toBe(true)
+        })
+    `,
```

Comment:
Could you ensure these are indenting to be at the same level as the object properties?

(Prettier won't change them because they're technically content)